scheduler/src/cook/mesos/api.clj

```diff
@@ -366,7 +366,34 @@
                           (::results ctx)))
       ring.middleware.json/wrap-json-params))
 
+(defn waiting-jobs
+  [mesos-pending-jobs-fn]
+  (-> (resource
+        :available-media-types ["application/json"]
+        :allowed-methods [:get]
+        :handle-ok (fn [ctx]
+                     (->> (mesos-pending-jobs-fn)
```

Comment:
This returns the current ordered queue of jobs whereas running just returns all jobs that are running in no particular order.